### ISSUE TITLE
Need help to build under Windows

### ISSUE THREAD

**Issue by bubnenkoff** [Groterik/qtcreator-dlangeditor#18]
Do I need Qt Creator sources to build this plugin?

I can't understand what Path I should to set on Windows. Now I done it's like:
qmake "QTC_SOURCE=C:\qtcreator_src\src" "QTC_BUILD=C:\qtcreator_src\src\libs"

QTC_SOURCE path looks fine, but what about QTC_BUILD?

**Comment by bubnenkoff** [comment]
I tried to build it, and got next errors: http://img.ctrlv.in/img/15/09/27/560836f90a7ed.png

**Comment by Groterik** [comment]
Unfortunately I don't have any windows machine now and this error tells me nothing. As far as I remember I didn't use any platform dependent code, so I think that the problem is in build environment. I will try to find windows machine soon. What versions of Qt and QtC do you use?

**Comment by bubnenkoff** [comment]
Qt is 5.5 what is QtC 3.5 if you would able to build it please add bin plugin to repository to get more people to test it, without long building process.

**Comment by zoujiaqing** [comment]
Windows 10 + QT 5.5 can use it? :)

**Comment by Groterik** [comment]
> if you would able to build it please add bin plugin to repository ...
Just added the link to the binaries at Bintray. Compiling on windows should also be fixed.
> Windows 10 + QT 5.5 can use it? :)
Yes, you can try win32 binaries at Bintray or build from the sources.

**Comment by michaelc37** [comment]
Groterik , can you check your win32 builds on Bintray for this project and the qtcreator dub-manager? Every win32 zip i download is corrupted .. i cannot extract anything.

**Comment by Groterik** [comment]
Yeah, some error while publishing artifacts using dpl under windows. Will continue to investigate tomorrow.

**Comment by Groterik** [comment]
We are waiting for [dpl's issue](https://github.com/travis-ci/dpl/issues/399) to be resolved.

**Comment by Groterik** [comment]
Dpl issue was fixed and released, so the publisher archives with binaries look good now, I can open them at least. Please check them, if you're still interested in trying this :) Sorry for huge delays here, troubles with spare time as usual.
